articles/database/migrations/2021_08_05_193220_create_posts_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreatePostsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('posts', function (Blueprint $table) {
+            $table->id();
+            $table->string('title');
+            $table->string('image');
+            $table->string('description');
+            $table->unsignedBigInteger('category_id')->nullable();
```

Comment:
@FareenaEhtesham there should be foreign keys here. Watch this video: https://www.youtube.com/watch?v=-77LnBSL2Zk

Also, the task was to create a `belongsToMany` relationship, I don't see it in the migrations